config/jobs/kubernetes/sig-scalability/sig-scalability-experimental-periodic-jobs.yaml

```diff
@@ -361,3 +361,200 @@ periodics:
       - --test-cmd-name=ClusterLoaderV2
       - --timeout=40m
       - --use-logexporter
+
+# max volumes per cluster test cases for ephemeral volumes
+- name: ci-kubernetes-storage-scalability-max-emptydir-vol-per-cluster-500
+  tags:
+  - "perfDashPrefix: storage-max-emptydir-vol-per-cluster-500"
+  - "perfDashJobType: storage"
+  interval: 1h
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+    preset-dind-enabled: "true"
+    preset-e2e-kubemark-common: "true"
+  annotations:
+    testgrid-dashboards: sig-scalability-experiments
+    testgrid-tab-name: max-emptydir-vol-per-cluster-500
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20190719-7cdf877-master
+      args:
+      - --repo=k8s.io/kubernetes=master
+      - --repo=k8s.io/perf-tests=master
+      - --root=/go/src
+      - --timeout=140
+      - --scenario=kubernetes_e2e
+      - --
+      - --cluster=kubemark-500
+      - --extract=ci/latest
+      - --gcp-master-size=n1-standard-4
+      - --gcp-node-image=gci
+      - --gcp-node-size=n1-standard-8
+      - --gcp-nodes=8
+      - --gcp-project=k8s-jenkins-blocking-kubemark
```

Comment:
It's not that easy - 500-kubemark is a non-trivial job (as you see it's using 68-cores) and you're addding 4 such jobs. They will clearly not fit this project quota (btw the configs wouldn't work because you're sharing the project, cluster name etc. between jobs, so they would be cleaning one each other).

In order to introduce those tests, we would first need to create a project pool with appropriate quota, and only then start using it here.
@krzysied @mm4tt - FYI

I think that instead we may want to wait for https://github.com/kubernetes/perf-tests/pull/677 to be designed and implemented and then create a single job for it. @mm4tt - thoughts?

Reply:
Sorry, missed the comment. I'd wait for the TestSuite (https://github.com/kubernetes/perf-tests/pull/677) to be implemented. I think the design is basically approved and implementation should be quite straightforward. @krzysied, how much time will you need to implement? 
Once we have a single test suite we should be able to find you a project where you can run it.

Reply:
Now that TestSuite is merged, should we refactor our tests to use this test suite? I imagine all the similar 1 node tests can be run together.

Reply:
It's not merged - we only agreed on how this should be done (only the doc is merged).

Reply:
Oh sorry, I didn't realize it was just the design doc.